src/phoenix/server/api/routers/v1/__init__.py

```diff
@@ -24,9 +28,31 @@ async def prevent_access_in_read_only_mode(request: Request) -> None:
         )
 
 
+async def authorize(request: Request) -> None:
```

Comment:
```suggestion
async def authentication(request: Request) -> None:
```

Reply:
while 401 is unauthorized it's an authentication status:

```
There's a problem with 401 Unauthorized, the HTTP status code for authentication errors. And that’s just it: it’s for authentication, not authorization. Receiving a 401 response is the server telling you, “you aren’t authenticated–either not authenticated at all or authenticated incorrectly–but please reauthenticate and try again.” To help you out, it will always include a WWW-Authenticate header that describes how to authenticate.
```